qtodotxt/ui/views/filters_tree_view.py

```diff
@@ -104,6 +120,7 @@ def _addDefaultTreeItems(self, tree):
             FilterTreeWidgetItem(None, ['Complete'], CompleteTasksFilter(), getIcon('x.png'))
         tree.addTopLevelItems([
             self._incompleteTasksItem,
+            self._dueItem,
```

Comment:
I don't really agree. In my opinion *Due* is related to time, just as *Pending*. *Uncategorized* is related to *Context* and *Projects* and should be placed before or after (above *Complete*).